driver/ppm_events_public.h

```diff
@@ -20,6 +20,168 @@ or GPL2.txt for full copies of the license.
 #include "../userspace/common/types.h"
 #endif
 
+/* DRIVERs FEATURE GATES:
```

Comment:
I like this feature gates! But what about moving these to their own header, like
`features.h`?

Reply:
yeah, that's a good idea, I will move them in a `features.h` file